packages/appeals-service-api/src/controllers/appeals.js

```diff
@@ -18,199 +17,71 @@ module.exports = {
     appeal.updatedAt = now;
 
     logger.debug(`Creating appeal ${appeal.id} ...`);
-    try {
-      await mongodb
-        .get()
-        .collection('appeals')
-        .insertOne({ _id: appeal.id, uuid: appeal.id, appeal });
 
+    const document = await insertAppeal(appeal);
+
+    if (document.result.ok) {
```

Comment:
` if (document.result && document.result.ok) {`

// or 

` if (document.result?.ok) {`

think we can use optional chaining now

? idk maybe im being anal